america/mexico/fiscal.source-spec.yaml

```diff
@@ -112,22 +122,24 @@ fields:
   options: {}
   osType: economic-classification:generic:level2:label
 
-- header: PARTIDA_GENERICA
+- header: ID_PARTIDA_GENERICA
+  title: Partida Genérica
   aliases:
   - "Partida Gen\xE9rica"
   options: {}
-  osType: economic-classification:generic:level3:code:part
+  osType: economic-classification:generic:level3:code:full
 - header: DESC_PARTIDA_GENERICA
   aliases:
   - "Descripci\xF3n de Partida Gen\xE9rica"
   options: {}
   osType: economic-classification:generic:level3:label
 
-- header: PARTIDA_ESPECIFICA
+- header: ID_PARTIDA_ESPECIFICA
```

Comment:
If you're changing the headers, then you also need to update this file:
https://github.com/Victordeleon/os-data-importers/blob/master/america/mexico/objeto-del-gasto.py

which calculates the objeto del gasto fields and assumes specific header names.